workspace/src/app/views/shared/RuleEditor/view/RuleEditorCanvas.tsx

```diff
@@ -500,56 +509,63 @@ export const RuleEditorCanvas = () => {
     };
 
     return (
-        <GridColumn full>
-            <ReactFlow
-                id={"ruleEditor-react-flow-canvas"}
-                data-test-id={"ruleEditor-react-flow-canvas"}
-                configuration={"linking"}
-                ref={reactFlowWrapper}
-                elements={modelContext.elements}
-                onElementClick={onElementClick}
-                onSelectionDragStart={handleSelectionDragStart}
-                onSelectionDragStop={handleSelectionDragStop}
-                onEdgeContextMenu={onEdgeContextMenu}
-                onElementsRemove={onElementsRemove}
-                onConnectStart={onConnectStart}
-                onConnect={onConnect}
-                onConnectEnd={onConnectEnd}
-                onNodeDragStart={handleNodeDragStart}
-                onNodeDragStop={handleNodeDragStop}
-                onNodeMouseEnter={onNodeMouseEnter}
-                onNodeMouseLeave={onNodeMouseLeave}
-                onNodeContextMenu={onNodeContextMenu}
-                onSelectionContextMenu={onSelectionContextMenu}
-                onSelectionChange={onSelectionChange}
-                onLoad={onLoad}
-                onDrop={onDrop}
-                onDragOver={onDragOver}
-                onEdgeUpdateStart={onEdgeUpdateStart}
-                onEdgeUpdateEnd={onEdgeUpdateEnd}
-                onEdgeUpdate={onEdgeUpdate}
-                connectionLineType={ConnectionLineType.Step}
-                snapGrid={snapGrid}
-                snapToGrid={true}
-                zoomOnDoubleClick={false}
-                maxZoom={1.25}
-                multiSelectionKeyCode={18} // ALT
-            >
-                <MiniMap
-                    flowInstance={reactFlowInstance}
-                    nodeClassName={minimapNodeClassName}
-                    nodeColor={minimapNodeColor}
-                    enableNavigation={true}
-                />
-                <Controls
-                    showInteractive={!!modelContext.setIsReadOnly}
-                    onInteractiveChange={(isInteractive) =>
-                        modelContext.setIsReadOnly && modelContext.setIsReadOnly(!isInteractive)
+        <>
+            {modelContext.showStickyNoteModal ? (
```

Comment:
As noted before, the sticky note modal can be much more "local" where the actual create/update action is fired.